src/SAWScript/X86.hs

```diff
@@ -405,6 +409,9 @@ translate opts elf fun =
   do let name = funName fun
      sayLn ("Translating function: " ++ BSC.unpack name)
 
+     -- TODO? do we need to pass in the mdMap into more places in this mode?
```

Comment:
I'm also unsure of this - AFAIK there's only one old proof currently using this mode, at https://github.com/GaloisInc/aes-gcm-siv-proof. This builds against an old SAW commit anyway. There's probably a conversation to be had about deprecating/removing this interface.

Reply:
If this isn't an interface that we intend to use for future proofs, then I probably won't worry to much about piping in new usability features.

Is proof something that, in principle, can be done with other methods now?

Reply:
OK, in that case, I'll leave this alone, and it probably is worth having a discussion about removing this interface.